### ISSUE TITLE
[proposal] Add setting to set align of column headers

### ISSUE THREAD

**Issue by lechu445** [shuGH/vscode-table-formatter#12]
It would be nice to have possibility to choose align of table headers. 
Sublime Table Editor plugin centers all table column headers by default, so this setting could improve compability with tables created with the plugin.

Examples:
center_column_headers=false:
```
| header          |
|-----------------|
| value           |
| some_long_value |

```
center_column_headers=true:
```
|      header      |
|------------------|
| value            |
| some_long_value1 |
```
center_column_headers=true:
```
|      header     |
|-----------------|
| value           |
| some_long_value |
```

**Comment by shuGH** [comment]
ok
i understood

**Comment by shuGH** [comment]
Thank you for waiting.
I released v1.2.1 and added center aligned header configuration.
Try `tableformatter.common.centerAlignedHeader = true`.

**Comment by lechu445** [comment]
Thank you!
